CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+* Fix deprecation warning. (#237)
```

Comment:
Is it worth a further sentence saying what the deprecation warning was?

Reply:
I don't know. Is it? 😄

Reply:
Done

Reply:
Sorry, I shouldn't phrase things as questions. 😅